modules/rostests/kmtests/ntos_ob/ObQuery.c

```diff
@@ -3,6 +3,7 @@
  * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:     Kernel mode tests for object information querying
  * COPYRIGHT:   Copyright 2023 George Bișoc <george.bisoc@reactos.org>
+ *              Copyright 2024 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
```

Comment:
```suggestion
 *              Copyright 2025 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
```
🤣

Reply:
ah no, code was written on the 30th of december, so it still was 2024 :joy: